docs/guides/prisma.md

```diff
@@ -0,0 +1,360 @@
+
+## Prerequisites
+
+To follow along with this guide, you will need to have the following:
+- [Node.js](https://nodejs.org/) >= v16.17.0 or >= v18.8.0
+- [npm](https://docs.npmjs.com/cli/) v7 or later
+- A code editor, for example [Visual Studio Code](https://code.visualstudio.com/)
+- A Platformatic DB project
+
+## Setup Prisma
+
+Install the [Prisma CLI](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-cli) and the [`db-diff`](https://github.com/ruheni/db-diff) development dependencies in your project:
+
+```bash
+npm install --save-dev prisma @ruheni/db-diff
+```
+
+Next, initialize Prisma in your project
+
+```bash
+npx prisma init
+```
+
+This command does the following:
+
+- Creates a new directory called `prisma` which contains a file called `schema.prisma`. This file defines your database connection and the Prisma Client generator.
+- Creates a `.env` file at the root of your project if it doesn't exist. This defines your environment variables (used for your database connection).
+
+You can specify your preferred database provider using the `--datasource-provider` flag, followed by the name of the provider: 
+
+```bash
+npx prisma init --datasource-provider mysql # or sqlite, sqlserver, cockroachdb
+```
+
+Prisma uses the `DATABASE_URL` environment variable to connect to your database to sync your database and Prisma schema. It also uses the variable to connect to your database to run your Prisma Client queries. 
+
+If you're using PostgreSQL, MySQL, SQL Server, or CockroachDB, ensure that the `DATABASE_URL` used by Prisma is the same as the one used by Platformatic DB project. 
+
+If you have an existing project, refer to the [Adding Prisma to an existing Platformatic DB project](#adding-prisma-to-an-existing-project) section. If you're adding Prisma to a new project, refer to the [Adding Prisma to a new project](#adding-prisma-to-an-new-project).
+
+## Adding Prisma to an existing project
+
+If you have a brownfield Platformatic DB project, you can introspect your database and generate the data model in your Prisma schema with the following command:
+
+```bash
+npx prisma db pull
+```
+
+The command will introspect your database and generate the [data model](https://www.prisma.io/docs/concepts/components/prisma-schema/data-model)
+
+Next, add the `@@ignore` attribute to the `versions` model to exclude it from the Prisma Client API:
+
+```diff
+model versions {
+  version BigInt    @id
+  name    String?
+  md5     String?
+  run_at  DateTime? @db.Timestamptz(6)
+
++  @@ignore
+}
+```
+
+To learn how you can evolve your database schema, you can jump to the [Evolving your database schema](#evolving-your-database-schema) section.
+
+## Adding Prisma to a new project
+
+Define a `Post` model with the following fields in your `schema.prisma` file:
+```groovy
+// prisma/schema.prisma
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgres"
+  url      = env("DATABASE_URL")
+}
```

Comment:
To keep things concise, I think these two blocks above can be removed as they will already exist in `schema.prisma`.